### ISSUE TITLE
Unable to get any command to work either either sol or ar keypairs

### ISSUE THREAD

**Issue by znebby** [CalebEverett/arloader#3]
Version: 0.1.35

Ubuntu WSL on Windows 10.

Upload Command:

./arloader upload "*.png" --log-dir "status/asset/"
error: The following required arguments were not provided:
    --ar-keypair-path <AR_KEYPAIR_PATH>

./arloader upload "*.png" --log-dir "status/asset/" --ar-keypair-path "./arweave-key"
error: Found argument '--ar-keypair-path' which wasn't expected, or isn't valid in this context
        Did you mean --sol-keypair-path?

./arloader upload "*.png" --log-dir "status/asset/" --sol-keypair-path "~/.config/solana/id.json" --with-sol
error: The following required arguments were not provided:
    --ar-keypair-path <AR_KEYPAIR_PATH>

./arloader upload "*.png" --log-dir "status/asset/" --sol-keypair-path "~/.config/solana/id.json" --with-sol --ar-keypair-path "./arweave-key"
error: Found argument '--ar-keypair-path' which wasn't expected, or isn't valid in this context
        Did you mean --sol-keypair-path?

Estimate Command:

./arloader estimate "*.png"
error: The following required arguments were not provided:
    --ar-keypair-path <AR_KEYPAIR_PATH>

./arloader estimate "*.png" --ar-keypair-path "./arweave-key"
error: Found argument '--ar-keypair-path' which wasn't expected, or isn't valid in this context

**Comment by CalebEverett** [comment]
Can you try using full paths - it doesn't like the `~`. I will validate for that in a future release. Can you try putting the keypair paths as environment variables? The command line parser may be expecting those as options to precede the subcommands. I will move those out as options so that they can be passed after the subcommands, e.g., estimate, in the next release.

**Comment by CalebEverett** [comment]
I'm going to close this since it seems like it's a matter of entering the commands in the order the parser expects, but feel free to open back up or add additional/comments questions if you still have trouble getting it going.

**Comment by znebby** [comment]
Thanks - I got it to work when putting the options in the correct order and using full paths without the ~.

However, I noticed you pushed out 0.1.36, which is meant to add support for ~ paths, but it's not working for me.

./arloader upload "*.png" --ar-keypair-path ./arweave-key --log-dir "status/asset/" --with-sol --sol-keypair-path "/home/user/.config/solana/id.json"
InsufficientSolFunds

Uploaded 0 KB in 0 files in 0 bundle transaction(s). Run `arloader update-status --log-dir "status/asset/"` to update statuses.


./arloader upload "*.png" --ar-keypair-path ./arweave-key --log-dir "status/asset/" --with-sol --sol-keypair-path "~/.config/solana/id.json"
error: Invalid value for '--sol-keypair-path <SOL_KEYPAIR_PATH>': Keypair path does not exist.
